packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/resume/accessories-resume.controller.js

```diff
@@ -21,7 +22,7 @@ export default class TelecomTelephonyLinePhoneAccessoriesResumeCtrl {
     };
 
     this.loading.init = true;
-    this.process = this.TucTelephonyAccessoriesOrderProcess.getOrderProcess();
+    this.processc = this.TucTelephonyAccessoriesOrderProcess.getOrderProcess();
```

Comment:
The variable had the wrong name ?

Reply:
Hmm @varun257 I see this as resolved, but there's a typo in the name ?